### PR TITLE
Audit pagination

### PR DESCRIPTION
**Pull request by GraceGardner**
## Description & motivation 💭 
Adds darkmode styling to the pagination and children components 

### Screenshots (if applicable) 📸 
Screenshot 2024-04-04 at 12 22 57 PM
Screenshot 2024-04-04 at 12 23 14 PM

### Design Considerations 🎨 
Needs approval from Ash 

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Issue(s) closed 
DT-1994

### CODE REVIEW

File: src/lib/holocene/select/filter-select.svelte [hunk 1]
```diff
@@ -34,7 +34,7 @@
   {id}
   bind:value={_value}
   {...$$props}
-  class="border-[1px]  outline-none focus-visible:outline focus-visible:outline-blue-700"
+  class="border-[1px] border-secondary bg-transparent outline-none focus-visible:outline focus-visible:outline-blue-700"
```
Comment (laurakwhit) [inline src/lib/holocene/select/filter-select.svelte:37]:
```suggestion
  class="border border-secondary bg-transparent outline-none focus-visible:outline focus-visible:outline-blue-700"
```

File: src/lib/holocene/select/simple-select.svelte [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
 <style lang="postcss">
   select {
-    @apply text-slate-500;
+    @apply text-primary;
```
Comment (laurakwhit) [inline src/lib/holocene/select/simple-select.svelte:57]: Might we want to add similar styling here as well?

```suggestion
    @apply text-primary border-secondary bg-transparent;
```

| Before | After |
|--|--|
|Screenshot 2024-04-05 at 10 41 03 AM|Screenshot 2024-04-05 at 10 42 02 AM|
|Screenshot 2024-04-05 at 10 41 17 AM|Screenshot 2024-04-05 at 10 41 38 AM|

File: src/lib/holocene/pagination.svelte [hunk 3]
```diff
@@ -206,7 +206,7 @@
     setActiveRowIndex={store.setActiveRowIndex}
   />
   <nav
-    class={`flex ${
+    class={`flex text-primary ${
```
Comment (laurakwhit) [inline src/lib/holocene/pagination.svelte:209]: Is `text-primary` needed in both places? Should we be setting it in the `FilterSelect` instead?

Comment (GraceGardner) [reply src/lib/holocene/pagination.svelte:209]: Oh that's a really great idea! Still getting used to this codebase for sure.

File: src/lib/holocene/select/simple-select.svelte [hunk 4]
```diff
@@ -54,7 +54,7 @@
 
 <style lang="postcss">
   select {
-    @apply text-slate-500;
+    @apply border-secondary bg-transparent text-primary;
```
Comment (laurakwhit) [inline src/lib/holocene/select/simple-select.svelte:57]: Let's make this what we currently have on the `FilterSelect` and that way we can just remove the class on the `FilterSelect` component. 

```suggestion
    @apply border-secondary bg-transparent text-primary outline-none focus-visible:outline focus-visible:outline-blue-700;
```

Comment (laurakwhit) [reply src/lib/holocene/select/simple-select.svelte:57]: Also, let's make the border `1px` (so replace `border-2` [here](https://github.com/temporalio/ui/blob/main/src/lib/holocene/select/simple-select.svelte#L36) with `border`).

File: src/lib/theme/plugin.ts [hunk 5]
```diff
@@ -111,7 +111,7 @@ const dark: Partial<Variables<keyof typeof variables>> = {
 const temporal = plugin(
   ({ addComponents, addBase }) => {
     addBase({
-      ':root': variables,
+      ':root': dark,
```
[the remark this answers is not in the document]
Comment (GraceGardner) [reply src/lib/theme/plugin.ts:114]: Oh my gosh! Sorry! Thank you!

**Pull request merged**